bikespace_dashboard/index.html

```diff
@@ -9,15 +9,23 @@
     <link rel="stylesheet" href="leaflet/leaflet.css" />
     <link rel="stylesheet" href="css/main.css" />
     <link rel="icon" type="image/x-icon" href="assets/bikespace_logo_sm.svg" />
+    <script>
+        if (window.location.host === "dashboard.bikespace.ca") { 
+        let s = document.createElement("script");
+        s.src = "https://eu.umami.is/script.js";
```

Comment:
The tracking code snippet generated on umami refers to us `https://us.umami.is/script.js` im not sure if it makes a difference, maybe we should keep it to the us.umami.is to keep it consistent with the setup on umami.